### ISSUE TITLE
Usage type for Completion Requests is missing

### ISSUE THREAD

**Issue by PedroHase** [openai/openai-node#17]
**Description**

According to [the docs](https://beta.openai.com/docs/api-reference/completions/create) a response to a completion request should have a `usage` property that allows to see how many tokens have been used for the request + response. Manually checking the response of `openai.createCompletion` also shows that the `usage` property exists in `response.data`:

```ts
const response = await openai.createCompletion({
    model: 'text-davinci-002',
    prompt: `<someprompt>`,
  });

console.log(response.data.usage)
```

However, the [`CreateCompletionResponse`](https://github.com/openai/openai-node/blob/71e21f9c48e865531064ccc191b9073698282655/api.ts#L489) type does not include `usage` and thus Typescript is throwing an error when trying to access `usage` in an openai completion response.

**Expected Behavior**

`openai` should have a type definition for `usage` in `CreateCompletionResponse` that allows to see & access the used tokens in a request.

**Comment by PedroHase** [comment]
As a quick work around, it is possible to add the `usage` type by overwriting the type definition with ones own type definition (e.g. a `global.d.ts` in `src`):

```ts
import 'openai';

declare module 'openai' {
  export interface CreateCompletionResponse {
    usage?: {
      prompt_tokens: number;
      completion_tokens: number;
      total_tokens: number;
    };
  }
}
```

This will allow to the used tokes in `response.data.usage`.

**Comment by DeveloperHarris** [comment]
Also encountered this issue as well.

**Comment by schnerd** [comment]
Thanks for reporting – Published a fix for this in 3.0.1!
